resolwe_bio/processes/support_processors/gff_to_gtf.yml

```diff
@@ -0,0 +1,36 @@
+# ============
+# Gff to gtf
+# ============
+
+---
+
+- slug: gff-to-gtf
+  name: Annotation (GFF3)
+  data_name: 'gff'
```

Comment:
Include input file name in the data_name. Maybe use `"Convert to GFF ('{{ src.file }}')"`